DiamondLightSource/python-artemis
# Blueapi: Use bluesky context for plans

Fixes #826

### To test:
1. Confirm tests still pass
2. Confirm you can `run_artemis` against the simulator, write to an endpoint and that this starts a plan. (you may need to comment out some of the devices that are not yet in s03)
